datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/policy/GetGrantedPrivilegesResolver.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.datahub.graphql.resolvers.policy;
+
+import com.datahub.authorization.AuthorizationManager;
+import com.datahub.authorization.ResourceSpec;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import com.linkedin.datahub.graphql.generated.GetGrantedPrivilegesInput;
+import com.linkedin.datahub.graphql.generated.Privileges;
+import com.linkedin.datahub.graphql.resolvers.EntityTypeMapper;
+import com.linkedin.metadata.authorization.PoliciesConfig;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.bindArgument;
+
+
```

Comment:
Minor: can we add quick javadoc to this class - i've been trying to add to resolvers

Reply:
done!